### DISCUSSION TITLE
Marquee effect for autoplay plugin (continuous motion)

### DISCUSSION THREAD

**Discussion by cbergen** [naver/egjs-flicking#767] [Ideas]
I would like an option for autoplay where the cells do not snap into position. This would be just like the continuous motion of the old marquee element. This would work great with the moveType: freeScroll option.

**Comment by malangfox** [comment]
Hello @cbergen.

This suggestion may be like adding behavior similar to [this demo](https://codesandbox.io/s/flicking-autoplay-marquee-289jqn?file=/src/App.tsx) in our AutoPlay plugin.
Our team will discuss adding behavior like this via an option in the AutoPlay plugin.

Thanks for suggesting a new feature for our component!
